src/setupTests.ts

```diff
@@ -0,0 +1,5 @@
+//FIXME workaround svg.panzoom.js import crash even though it's not used
```

Comment:
I had the same issue in pypowsybl-jupyter, the problem is that svg.panzoom.js is somewhat broken.
A solution is to import the ESM module svg.panzoom.esm.js instead. This can be configured in jest config file.

Reply:
See that issue for more background :

https://github.com/svgdotjs/svg.panzoom.js/issues/82

Reply:
Thanks. We already use that hack (global SVG with setupTests) in Gridstudy, we gonna do the same here. I use the jsdom environnement for tests, not the node.js one. I don't know if it would work.

Reply:
Indeed, it actually requires jsdom to work since svg.js works on the DOM.
It would allow svg.js to actually modify the DOM, in the unit test.